src/hooks/common/useTokenRefresh.ts

```
@@ -1,46 +1,33 @@
-        }
+    const tokenRefresh = async () => {
+      if (accessToken) return;
+
+      try {
+        const newAccessToken = await getRefreshToken();
+
+        dispatch(tokenActions.setToken(newAccessToken));
+        axiosInstance.defaults.headers.Authorization = `Bearer ${newAccessToken}`;
+      } catch (error) {
+        store.dispatch(tokenActions.deleteToken());
+        delete axiosInstance.defaults.headers.Authorization;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)